src/uas/UAS.cc

```diff
@@ -2827,21 +2825,6 @@ void UAS::enableHilXPlane(bool enable)
         }
         qDebug() << "CREATED NEW XPLANE LINK";
         simulation = new QGCXPlaneLink(this);
-
-        float noise_scaler = 0.1f;

```

Comment:
All of the predefined aircraft work for me though of late I have been using a custom aircraft fie that we generated.  

The Rascal does experience stalls on my end.  Will post a log showing the behaviour when I get the bandwidth to run a sim.  Can you do the same showing the Rascal approaching stall and never achieving it.